tests/sparsity/rb/test_algo.py

```diff
@@ -158,7 +158,7 @@ def test_sparse_algo_can_calc_sparsity_rate__for_2_conv_model():
     assert compression_ctrl.sparsity_rate_for_model == (
         1 - (model.nz_weights_num + model.nz_bias_num) / (model.weights_num + model.bias_num)
     )
-    assert compression_ctrl.sparsity_rate_for_sparsified_modules == 1 - model.nz_weights_num / model.weights_num
+    assert compression_ctrl.sparsity_rate_for_sparsified_modules() == 1 - model.nz_weights_num / model.weights_num
 
```

Comment:
Should add a test that tests the optional parameter of the `sparsity_rate_for_sparsified_modules`.

Reply:
Done.